src/lib.rs

```diff
@@ -16,9 +16,10 @@ use std::path::Path;
 use std::io::{self, Read};
 
 mod parser;
+mod generator;
```

Comment:
Because there's only one thing this library can parse (markdown), but two things it can generate: html and markdown. :)